src/richie/apps/courses/templates/courses/cms/blogpost_detail.html

```diff
@@ -172,6 +158,23 @@ <h2 class="blogpost-glimpse-list__title"><span>{% trans 'Related posts' %}</span
                                 {% endfor %}
                             </div>
                         {% endblock related_blogposts %}
+
+                        {% block related_categories %}
+                          <div class="blogpost-glimpse-list">
+                            <h2 class="blogpost-glimpse-list__title"><span>{% trans 'Other categories of article' %}</span></h2>
+                            {% get_related_category_pages request.current_page.parent_page.get_child_pages as categories %}
+                            {% if categories %}
+                              <div class="p-0 category-tag-list category-tag-list--primary">
+                                <div class="category-tag-list__container">
+                                  <p class="offscreen">{% trans "Categories" %}</p>
```

Comment:
@Leimi do we still need this now that we have a title to the section?

Reply:
Good catch, yeah it should be removed as the title is there right before :+1: